django/apps/registry.py

```diff
@@ -10,6 +10,13 @@
 from .config import AppConfig
 
 
+class Tags(object):
```

Comment:
Sorry, you misinterpreted what I meant for this. I was thinking of adding a `checks_tag = 'contrib.sessions'` attribute to `SessionsConfig` in `django/contrib/sessions/apps.py`. That might be overly complicated though. How about just a `TAG = 'contrib.sessions'` module level constant in ` django/contrib/sessions/checks.py`? The idea is that since sessions is a contrib app, it shouldn't have implementation details outside of the application.